src/lib/i18n/i18n-sanitize.js

```diff
@@ -64,7 +83,7 @@ export function sanitize(statics, ...params) {
         });
 
       // If link has href and external origin => force rel and target
-      if (node.tagName === 'A' && node.getAttribute('href') != null) {
+      if (isLinkNodeWithHref(node)) {
```

Comment:
**suggestion:** I've done some research because it's not the first time I face this situation. Here's what I got:
```suggestion
      if (node instanceof HTMLAnchorElement) {
```

WDYT?